src/NBS.Appointments.Service/Controllers/AvailabilityController.cs

```diff
@@ -60,9 +61,21 @@ public async Task<IActionResult> Days([FromBody] AvailabilityByDayRequest reques
 
         [HttpPost]
         [Route("hours")]
-        public async Task<IActionResult> Hours([FromBody] AvailabilityByHourRequest request)
+        public Task<IActionResult> Hours([FromBody] SiteAvailabilityRequest request)
+        {            
+            return LookupAvailability(request, (qflowResponse) => Ok(AvailabilityHourResponse.FromQflowResponse(qflowResponse, request.Service, request.Date, DateTime.UtcNow)));
```

Comment:
out of interest, how does this handle an exception in the LookupAvailability method?